com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -27,9 +26,13 @@ public sealed class NetworkObject : MonoBehaviour
 #if UNITY_EDITOR
         private void OnValidate()
         {
-            if (UnityEditor.EditorApplication.isPlayingOrWillChangePlaymode && !NetworkManager.IsTestRun)
+            if (UnityEditor.EditorApplication.isPlaying && !string.IsNullOrEmpty(gameObject.scene.name))
```

Comment:
Could you add some comments explaining why we're early-outing in both of these cases?

Reply:
we already have tests covering this and the tests found this issue are on the way here: #754

Reply:
it's very tricky to comment these flows inline here, I might try.

Reply:
I tried 😛